tests/integration/targets/gitlab_project_members/tasks/main.yml

```diff
@@ -10,16 +10,19 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 - name: Install required library
-  pip:
+  ansible.builtin.pip:
     name: python-gitlab
     state: present
+    extra_args: >-
+      --break-system-packages
 
 - name: Clean UP before tests
   community.general.gitlab_project_members:
     api_url: "{{ gitlab_server_url }}"
     api_token: "{{ gitlab_api_access_token }}"
     project: "{{ gitlab_project }}"
     gitlab_user: "{{ username }}"
+    access_level: "{{ gitlab_access_level }}"
```

Comment:
Why are you adding this parameter along a `state=absent` ? I don't think it makes much sense to pass an access level during a removal operation. Could you please provide some rationale here?

Reply:
Thanks for clarifying. I just raised #9966 to investigate that.